odoo/tests/common.py

```diff
@@ -2060,10 +2060,16 @@ def start_tour(self, url_path, tour_name, step_delay=None, **kwargs):
             'keepWatchBrowser': kwargs.get('watch', False),
             'debug': kwargs.get('debug', False),
             'startUrl': url_path,
+            'delayToCheckUndeterminisms': kwargs.pop('delay_to_check_undeterminisms', 0),
```

Comment:
not a big fan of the option or the name, but I'm not a good namer either :)
I think in the end we'd want the runbot to run that in a nightly or something, so a CLI argument ? or some kind of odoorc hidden parameter ?
what do you think @Xavier-Do ?

Reply:
Your name will be mine